_NORTH_MAINTENANCE.md

```diff
@@ -0,0 +1,78 @@
+# Maintaing this recipe
+
+## Recipe Version Support
+
+| Qt Version    | Upstream Branch   | North Branch |
+| ------------- |:----------------- |:-------------|
+| 5.12.5        | stable/5.12.5     | north-1     |
+
+## Submitting changes
+
+You can fork the fork of the thalmic conan-qt fork if you want.  But it seems simpler to just push your changes to a a branch in the thalmic fork and PR from there.
+
+> Note: Please delete your branch as part of merging your PR
+
+## Updating Recipes/Qt Version
+
+1. `git remote add upstream git@github.com:bincrafters/conan-qt`
+
+> Add conan-qt as an upstream
+
+2. `git fetch upstream`
+3. `git checkout -b north-2 upstream/stable/5.13.2`
+
+> create a new branch from the upstream target branch.  In this example, we want to upgrade to qt 5.13.2
+
+4. `git push origin north-2`
+
+> push the new branch to the remote (NOTE: this is a vanilla branch)
+
+5. `git rebase north-1`
+
+> rebase the old branch with north changes onto the newest upstream branch
+
+> Note: the Qt 5.12.5 branch does not have a common ancestor to mainline developing so any North changes will need to be cherry-picked in `north-2`
+
+6. update the version in conanfile.py to match the branch version
+
+7. Discover and update any external dependencies following the instructions below
+
+## External Dependencies
+
+Building dependss on the following external depenendices that must be imported into North's conan-local remote.  When these dependencies change, you'll need to run the import team city jobs to import the updated dependencies below:
```

Comment:
```suggestion
Building depends on the following external dependencies that must be imported into North's conan-local remote.  When these dependencies change, you'll need to run the import team city jobs to import the updated dependencies below:
```

Reply:
I had to submit another change so I made the change as part of that.